src/aspire/utils/relion_interop.py

```diff
@@ -1,11 +1,92 @@
-class RlnOpticsGroup:
-    def __init__(self, df_row):
+import logging
+from collections import OrderedDict
+
+import numpy as np
+
+from aspire.storage import StarFile
+
+logger = logging.getLogger(__name__)
+
+
+# The metadata_fields dictionary below specifies default data types
+# of certain key fields used in the codebase,
+# which are originally read from Relion STAR files.
+relion_metadata_fields = {
```

Comment:
Do you for-see any reason developers or users might want this at the second import level in the future?

Reply:
Added to the `__init__` thanks